README.md

```diff
@@ -54,6 +38,11 @@ cd ..
 # If working with multiple SDKs, using a SDK-specific name is helpful.
```

Comment:
Not part of this PR but this line feels a little unclear and maybe unnecessary.

Reply:
I think "git clone --recursive" makes is unnecessary to go into the folder and run "git submodule update --init --recursive" so we can probably cut out lines 49 - 52. Unless there's some reason why the submodules wouldn't be properly updated with the initial "git clone --recursive" command.